src/main/kotlin/uk/gov/dluhc/printapi/config/DataRetentionConfiguration.kt

```diff
@@ -7,5 +7,5 @@ import java.time.Duration
 @ConfigurationProperties(prefix = "api.print-api.retention.period")
 @ConstructorBinding
 data class DataRetentionConfiguration(
-    val certificateDeliveryInfo: Duration
+    val certificateInitialRetentionPeriod: Duration
```

Comment:
Naming is hard, but hopefully there is sufficient kdoc where this is used (plus in the `Certificate` entity wrt `initialRetentionRemovalDate`) to explain what this means.

Reply:
💯 👍 
My only comment here would be re: class type. I think the variable name is accurate in that its a period of time
There are 2 classes related to this - `java.time.Duration` and `java.time.Period` , but the variable name doesnt match the class used
I don't know the difference between a `Duration` and a `Period` - and I suspect changing the class type down the stack to a `Period` might be a pain?, so maybe the variable name is a `certificateInitialRetentionDuration` ??

I don't know - like you said, naming is hard!

I tell you what, let's pretend I never mentioned this, and we'll leave it well alone !! 😁 👍 🤣

Reply:
A valid observation. Changed to `Period` as it's better suited anyway (since we're not interested in the time, only the number of days).